spec.md

```diff
@@ -212,12 +221,12 @@ that contains both context and data).
   * If present, MUST contain at least one entry
 * Examples:
   * authorization data
-  * content type
 
 ### data
 * Type: Arbitrary payload
 * Description: The event payload. The payload depends on the event-type,
-  schema-url and event-type-version.
+  schema-url and event-type-version, the pyload is encoded into a media format 
```

Comment:
s/pyload/payload/

Reply:
you may want to make this more normative, something like:
If the `content-type` attribute is specified then the `data`
MUST be encoded per the rules defined by the specified format.